beacon_node/network/src/network_beacon_processor/gossip_methods.rs

```diff
@@ -599,6 +601,67 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         }
     }
 
+    pub async fn process_gossip_data_column_sidecar(
+        self: &Arc<Self>,
+        message_id: MessageId,
+        peer_id: PeerId,
+        _peer_client: Client,
+        subnet_id: DataColumnSubnetId,
+        column_sidecar: Arc<DataColumnSidecar<T::EthSpec>>,
+        seen_duration: Duration,
+    ) {
+        let slot = column_sidecar.slot();
+        let block_root = column_sidecar.block_root();
+        let index = column_sidecar.index;
+        let delay = get_slot_delay_ms(seen_duration, slot, &self.chain.slot_clock);
+        // Log metrics to track delay from other nodes on the network.
+        metrics::set_gauge(
+            &metrics::BEACON_DATA_COLUMN_DELAY_GOSSIP,
+            delay.as_millis() as i64,
+        );
```

Comment:
This should be an observation of a histogram not a gauge, same for `BEACON_DATA_COLUMN_DELAY_GOSSIP_VERIFICATION`

Reply:
This was changed for block and blobs in #5510 and I made the same change here for consistency.